FRINXio/frinx-frontend
# Fd 526 wf builder save

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-526` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | Yes


HOW TO TEST:

try create/edit and save workflow and other functionality clone, export... should work
